Sources/Hummingbird/Router/ResponseGenerator.swift

```diff
@@ -13,14 +13,22 @@
 //===----------------------------------------------------------------------===//
 
 import HTTPTypes
+import HummingbirdCore
 
 /// Object that can generate a `Response`.
 ///
 /// This is used by `Router` to convert handler return values into a `Response`.
+#if compiler(>=6.2)
+public protocol ResponseGenerator: SendableMetatype {
```

Comment:
```suggestion
public protocol ResponseGenerator: _HB_SendableMetatype{
```

Shouldn't this use the typealias/protocol like the rest?

Reply:
You can't do that because it is used in conditional conformances. eg `extension Optional: ResponseGenerator where Wrapped: ResponseGenerator {}` and in those cases it then pre 6.2 it expects me to include the SendableMetatype conformance but 6.2 complains about an unnecessary repeated conformance if I do include it.